hello-1/test_package/conanfile.py

```diff
@@ -1,14 +1,16 @@
-from conans import ConanFile
+from conans import ConanFile, tools
 import os
 
 class PkgTestConan(ConanFile):
-    requires = "hello/0.1@armin/testing" % (username, channel)
+    requires = "hello/0.1@armin/testing"
     generators = "qmake"
+    settings = "os", "compiler", "arch", "build_type"
 
     def build(self):
-        os.chdir(self.conanfile_directory)
```

Comment:
Not necessary to change dir, the test_package functionality will ensure that the temp files goes to the right temp folder